gc/base/ParallelDispatcher.hpp

```diff
@@ -35,13 +35,18 @@
 #include "modronbase.h"
 
 #include "BaseVirtual.hpp"
-#include "Dispatcher.hpp"
 #include "EnvironmentBase.hpp"
 #include "GCExtensionsBase.hpp"
 
 class MM_EnvironmentBase;
 
-class MM_ParallelDispatcher : public MM_Dispatcher
+/**
+ * @todo Provide define documentation
+ * @ingroup GC_Base_Core
+ */
+#define J9MODRON_HANDLE_NEXT_WORK_UNIT(envPtr) envPtr->_currentTask->handleNextWorkUnit(envPtr)
```

Comment:
I know this was in Dispatcher, but semantically it belongs more to (Parallel)Task - it's just a macro-wrapper of Task's method handleNextWorkUnit. Try to move it there.